x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/process/writer/LengthEncodedWriter.java

```diff
@@ -27,18 +31,50 @@
  */
 public class LengthEncodedWriter implements RecordWriter {
     private OutputStream outputStream;
+    private OutputStream fileOutputStream;
```

Comment:
it might be nice to add some comment about what the difference between these two objects is.

Reply:
Added a comment.